tests/ci/codebuild/android/run_android_fips.yml

```diff
@@ -0,0 +1,8 @@
+version: 0.2
+
+phases:
+  build:
+    commands:
+      - chmod +x ./tests/ci/android/AWSLCAndroidTestRunner/gradlew
+      - cd tests/ci/android/AWSLCAndroidTestRunner
+      - ./gradlew assembleDebug assembleAndroidTest -PFIPS
```

Comment:
* I may miss something. We don't need to call any AWS CLI(device farm) to start the project? These seems to run the logics in the Docker image.
* nit: Move these logics to some bash script like other yml files?